.env.sample

```diff
@@ -0,0 +1,7 @@
+DB_USERNAME=
+DB_PASSWORD=
+DB_TEST=
+DB_HOST=
+DB_PORT=
+DB_DATABASE=
+DB_DEV=
```

Comment:
Make the environment variables more descriptive. e.g. `DB_CONFIG_PASSWORD` instead of `DB_PASSWORD`. Also, add a description to the environment variables. .e.g.: `DB_CONFIG_USERNAME=Your database's username`